docs/rich_text_editor.rst

```diff
@@ -0,0 +1,41 @@
+Rich Text Editor
```

Comment:
Maybe add a warning, that reverting back to plain text will not remove the HTML text automatically.